client/packages/common/src/intl/locales/fr/replenishment.json

```diff
@@ -1,103 +1,104 @@
-    "messages.confirm-delete-requisition-lines_one": "Cela supprimera définitivement 1 ligne de cette commande",
-    "messages.confirm-delete-requisition-lines_other": "Cela supprimera définitivement {{count}} lignes de cette commande",
-    "messages.confirm-delete-requisitions_one": "Cette commande interne sera définitivement supprimée",
-    "messages.confirm-delete-requisitions_other": "Cela supprimera définitivement {{count}} commande interne",
-    "messages.confirm-delete-shipment-lines_one": "Cela supprimera définitivement 1 ligne du bon de livraison",
-    "messages.confirm-delete-shipment-lines_other": "Cela supprimera définitivement {{count}} lignes du bon de livraison",
-    "messages.confirm-delete-shipments_one": "Ce bon de livraison sera définitivement supprimé",
-    "messages.confirm-delete-shipments_other": "Cela supprimera définitivement {{count}} bons de livraison",
-    "messages.could-not-save": "Impossible de sauvegarder",
-    "messages.deleted-lines": "{{number}} ligne(s) supprimée(s)",
-    "messages.deleted-lines_one": "{{count}} ligne supprimée",
-    "messages.deleted-lines_other": "{{count}} lignes supprimées",
-    "messages.deleted-requisitions": "{{number}} réquisition(s) supprimée(s)",
-    "messages.deleted-requisitions_one": "{{count}} commande supprimée",
-    "messages.deleted-requisitions_other": "{{count}} commandes supprimées",
-    "messages.deleted-shipments_one": "{{count}} bon de livraison supprimé",
-    "messages.deleted-shipments_other": "{{count}} bons de livraison supprimées",
-    "messages.inbound-shipment-created-on": "Bon de livraison créé le {{date}}",
-    "messages.internal-order-created-on": "Commande interne créée le {{date}}",
-    "messages.no-related-documents": "Aucun document connexes",
-    "messages.no-shipments-yet": "Aucun bon de livraison n'a encore été créé.",
-    "messages.off-hold-confirmation": "Cela permettra de réactiver les changements de statut",
-    "messages.on-hold-confirmation": "Cela empêchera tout changement de statut jusqu'au déverrouillage",
-    "messages.requested-to-suggested": "Cela définira les quantités demandées avec les quantités suggérées.",
-    "messages.select-rows-to-delete": "Sélectionner des lignes pour les supprimer",
-    "messages.select-rows-to-delete-them": "Sélectionner les lignes pour les supprimer",
-    "messages.confirm-inbound-status-as": "Confirmer le status comme $t({{status}}) ?\nNote : changer le statut va supprimer les lignes avec zéro quantité.",
-    "messages.error-saving-shipment": "Erreur lors de la sauvegarde du bon de livraison 🥺",
-    "messages.shipment-saved": "Bon de livraison sauvegardé 🥳",
-    "messages.confirm-status-as": "Confirmer le status comme $t({{status}})",
-    "messages.saved": "Sauvegardé",
-    "heading.tax": "Taxe",
-    "heading.grand-total": "Grand total",
-    "heading.edit-tax-rate": "Modifier le taux de taxation (%)"
+  "button.add-from-master-list": "Utiliser une liste",
```

Comment:
not sure what's changed here

Reply:
Just the addition of `template.requisition-sent` at the end.. and the usual prettier changes on touching the french version